gtsfm/two_view_estimator.py

```diff
@@ -215,6 +215,68 @@ def bundle_adjust(
 
         return i2Ti1_optimized.rotation(), Unit3(i2Ti1_optimized.translation()), valid_corr_idxs
 
+    def __get_2view_report_from_results(
+        self,
+        i2Ri1_computed: Optional[Rot3],
+        i2Ui1_computed: Optional[Unit3],
+        keypoints_i1: Keypoints,
+        keypoints_i2: Keypoints,
+        verified_corr_idxs: np.ndarray,
+        inlier_ratio_wrt_estimate: float,
+        gt_camera_i1: Optional[gtsfm_types.CAMERA_TYPE],
+        gt_camera_i2: Optional[gtsfm_types.CAMERA_TYPE],
+        gt_scene_mesh: Optional[Any],
+    ) -> TwoViewEstimationReport:
+        """Generate a TwoViewEstimationReport from the results of the two-view estimation.
+
+        Currently metrics wrt GT camera only supports cases where gt_camera is PinholeCameraCal3Bundler.
+
+        Args:
+            i2Ri1_computed: computed relative rotation.
```

Comment:
nit: Capitalization for google style

Reply:
copilot would write in lowercase, probably because all other comments in this file used lowercase, Ive updated them all.